examples/gluon_character_cnn/README.md

```diff
@@ -0,0 +1,144 @@
+# Character-level CNN Model in Gluon trained using Amazon Product Dataset
+
+In this example, we show how to create a service which classifies a review into product type using [Character-level Convolutional Network Model (CNN) model](https://papers.nips.cc/paper/5782-character-level-convolutional-networks-for-text-classification.pdf) model by Yann LeCunn. This model is trained on [Amazon product data](http://jmcauley.ucsd.edu/data/amazon/) and training detail can be found in a detailed tutorial from Thomas Delteil on [Character CNN training.](https://github.com/ThomasDelteil/CNN_NLP_MXNet).
+
+
+# Step by step to create service
+
+## Step 1 - Download the Gluon Char CNN model file, signature file and model parameter file.
```

Comment:
There is no mention of synset download.

Reply:
The code works as-is, will update the comments